enums.go

```diff
@@ -0,0 +1,282 @@
+
+//5.11 Playback Cessation Modes
+
+const (
+	EPlaybackCessationModesOnVideoCompletion int8 = iota + 1
+	EPlaybackCessationModesOnLeavingViewport
+	EPlaybackCessationModesOnLeavingViewportContinuesAsFloatingSlider
+)
+
+//5.12 Start Delay
+
+const (
+	EStartDelayGenericPostRoll int8 = iota - 2
+	EStartDelayGenericMidRoll
+	EStartDelayPreRoll
+)
+
+//5.13 Production Quality
+
+const (
+	EProductionQualityUnknown int8 = iota
+	EProductionQualityProfessionallyProduced
+	EProductionQualityProsumer
+	EProductionQualityUserGenerated
+)
+
+//5.14 Companion Types
+
+const (
+	ECompanionTypesStaticResource int8 = iota + 1
+	ECompanionTypesHTMLResource
+	ECompanionTypesIframeResource
+)
+
+//5.15 Content Delivery Methods
+
+const (
+	EContentDeliveryMethodsStreaming int8 = iota + 1
+	EContentDeliveryMethodsProgressive
+	EContentDeliveryMethodsDownload
+)
+
+//5.16 Feed Types
+
+const (
+	EFeedTypesMusicService int8 = iota + 1
+	EFeedTypesFMAMBroadcast
+	EFeedTypesPodcast
+)
+
+//5.17 Volume Normalization Modes
+
+const (
+	EVolumeNormalizationModesNone int8 = iota
+	EVolumeNormalizationModesAdVolumeAverageNormalizedToContent
+	EVolumeNormalizationModesAdVolumePeakNormalizedToContent
+	EVolumeNormalizationModesAdLoudnessNormalizedToContent
+	EVolumeNormalizationModesCustomVolumeNormalization
+)
+
+//5.18 Content Context
+
+const (
+	EContentContextVideo int8 = iota + 1
+	EContentContextGame
+	EContentContextMusic
+	EContentContextApplication
+	EContentContextText
+	EContentContextOther
+	EContentContextUnknown
+)
+
+//5.19 IQG Media Ratings
+
+const (
+	EIQGMediaRatingsAllAudiences int8 = iota + 1
+	EIQGMediaRatingsEveryoneOver12
+	EIQGMediaRatingsMatureAudiences
+)
+
+//5.20 Location Type
+
+const (
+	ELocationTypeGPS int8 = iota + 1
+	ELocationTypeIP
+	ELocationTypeUser
+)
+
+//5.21 Device Type
+
+const (
+	EDeviceTypeMobileTablet int8 = iota + 1
+	EDeviceTypePersonalComputer
+	EDeviceTypeConnectedTV
+	EDeviceTypePhone
+	EDeviceTypeTablet
+	EDeviceTypeConnectedDevice
+	EDeviceTypeSetTopBox
+)
+
+//5.22 Connection Type
+
+const (
+	EConnectionTypeUnknown int8 = iota
+	EConnectionTypeEthernet
+	EConnectionTypeWIFI
+	EConnectionTypeCellularNetworkUnknownGeneration
+	EConnectionTypeCellularNetwork2G
+	EConnectionTypeCellularNetwork3G
+	EConnectionTypeCellularNetwork4G
+)
+
+//5.23 IP Location Services
+
+const (
+	EIPLocationServiceIP2Location int8 = iota + 1
+	EIPLocationServiceNeustar
+	IPLocationServiceMaxMind
+	IPLocationServiceNetAcuity
+)
+
+//5.24 No-Bid Reason Codes
+
+const (
+	ENoBidReasonUnknownError int8 = iota
+	ENoBidReasonCodeTechnicalError
+	ENoBidReasonCodeInvalidRequest
+	ENoBidReasonCodeKnownWebSpider
+	ENoBidReasonCodeSuspectedNonHumanTraffic
+	ENoBidReasonCodeCloudDataCenterProxyIP
+	ENoBidReasonCodeUnsupportedDevice
+	ENoBidReasonCodeBlockedPublisherOrSite
+	ENoBidReasonCodeUnmatchedUser
+	ENoBidReasonCodeDailyReaderCapMet
+	ENoBidReasonCodeDailyDomainCapMet
+)
+
+//5.25 Loss Reason Codes
+
+const (
+	ELossReasonCodesBidWon int8 = iota
```

Comment:
So, you used `ENoBidReasonCode` in prev enum, but here you use `ELossReasonCodes` (plural). Naming should be consistent.